internal/backends/postgresql/metadata/registry.go

```diff
@@ -16,20 +16,36 @@ package metadata
 
 import (
 	"context"
+	"fmt"
```

Comment:
github collapses this file, majority of change is in this file, please review it.

Reply:
That comment is also collapsed. But it is possible to leave a comment on the file now:

![CleanShot 2023-09-20 at 12 29 42@2x](https://github.com/FerretDB/FerretDB/assets/11512/d744d01a-b1e0-4e3b-b51a-994baa8e3a8d)

Reply:
:scream:

That's a nice feature I didn't know!